cookiecutter.json

```diff
@@ -42,6 +42,6 @@
   "use_heroku": "n",
   "use_travisci": "n",
   "keep_local_envs_in_vcs": "y",
-
+  "use_pre_commit": "n",
```

Comment:
Looking at the diff now, I'm no longer sure we should make it an option, it's adding complexity to the template for not much benefit. Let's make it the default and mention it in our README, [under the list of features](https://github.com/pydanny/cookiecutter-django/blob/master/README.rst#features).

Reply:
You mean that `pre_commit` exists in final project as default and mention that, is that right?

Reply:
Yes